soc/riscv/riscv-privileged/andes_v5/Kconfig.soc

```diff
@@ -28,6 +28,12 @@ config RV32I_CPU
 	select RISCV_ISA_EXT_ZICSR
 	select RISCV_ISA_EXT_ZIFENCEI
 
+config RV32E_CPU
```

Comment:
We already have `CONFIG_RISCV_ISA_RV32E`, why this is needed?

Reply:
AE350 is FPGA based platform and we want RV32E can be selected by 
menuconfig or board_defconfig for AE350 bitstream integrate with RV32E core.